### PR TITLE
explain how to run installer targeting different architectures

### LINKED ISSUE #2840: Improve documentation for cross-platform builds
Please provide information requested in the template below as much as possible.

#### Describe the bug

The instructions in README.md is not clear about how to do cross-platform builds (in my case, building an x86_64 image on a M1 Mac laptop)

From #eve-help it was revealed that a build needs to be done using a special docker image tag

#### Recent release

Tested with lfedge/eve:latest (0e33e031940c)

#### How to reproduce

Initial (not working) attempt:
(on M1 laptop)
docker run --rm lfedge/eve installer_raw > installer.raw
Trying to boot this via qemu-system-x86_64 fails

I also tried building a GCP image (docker run --rm lfedge/eve -f gcp installer_raw > eve.tar.gz). Booting this in GCE gives "not a bootable image)

Doing the same steps from an x86_64 machine worked fine.

From #eve-help, the right invocation when cross-building is:
docker run lfedge/eve:8.11.0-kvm-amd64 installer_raw

### PR DESCRIPTION
**Pull request by deitch**
Fixes #2840 

Makes it clear in the README how to run the installer for different architectures.

cc @avx-rchung

### CODE REVIEW

**Comment by deitch** [comment]
Yeah, we probably should. I will add something for it.

**Comment by deitch** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -38,6 +38,49 @@ EVE Docker container `lfedge/eve:<version>` is used to produce these deployment
...
+* hypervisor: `kvm`, `xen`, `mini`
+
+Note that not all hypervisors are supported on all architectures.
+
+For example:
+
+* `docker run lfedge/eve:8.11.0-kvm-arm64`: installer for 8.11.0 using kvm on arm64
+* `docker run lfedge/eve:8.11.0-xen-arm64`: installer for 8.11.0 using xen on arm64
+* `docker run lfedge/eve:8.11.0-xen-amd64`: installer for 8.11.0 using xen on amd64
+* `docker run lfedge/eve:8.11.0-mini-riscv64`: installer for 8.11.0 using mini on riscv64
```
Comment (eriknordmark) [inline README.md:54]: None of these get you the installer; they just print a usage string. Did you intend to add "installer" to the line?

File: README.md [hunk 2]
```diff
@@ -38,6 +38,49 @@ EVE Docker container `lfedge/eve:<version>` is used to produce these deployment
...
+
+If you leave off the architecture it will default to whatever architecture you
+are running on.
+
+If you just use `snapshot` or `latest`, it will default to the architecture you are running on and the `kvm` hypervisor.
+
+For example:
+
+* `docker run lfedge/eve:8.11.0-kvm`: installer for 8.11.0 using kvm on your architecture
+* `docker run lfedge/eve:latest`: installer for latest image using kvm on your architecture
```
Comment (eriknordmark) [inline README.md:64]: Ditto

File: README.md [hunk 3]
```diff
@@ -38,6 +38,49 @@ EVE Docker container `lfedge/eve:<version>` is used to produce these deployment
...
+
+Note that each docker image is built for the architecture for which it runs. This `lfedge/eve:8.11.0-kvm-arm64` not only
```
Comment (eriknordmark) [inline README.md:66]: Ditto

File: README.md (hunk 1 above)
Comment (deitch) [reply README.md:54]: Sort of. There are different installer commands (`installer_raw`, `installer_net`, etc.), so I didn't want to pick. I can add something that indicates a command should be there.

**Comment by deitch** [comment]
No idea why the unit tests failed; this just updated the README 🤷‍♂️

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**